library/src/main/java/com/xwray/groupie/GroupAdapter.java

```diff
@@ -107,30 +98,44 @@ public void updateAsync(@NonNull final List<? extends Group> newGroups) {
      *
      * @param newGroups List of {@link Group}
      * @param onAsyncUpdateListener Optional callback for when the async update is complete
+     * @param detectMoves Boolean is passed to {@link DiffUtil#calculateDiff(DiffUtil.Callback, boolean)}
      */
     @SuppressWarnings("unused")
-    public void updateAsync(@NonNull final List<? extends Group> newGroups, @Nullable final OnAsyncUpdateListener onAsyncUpdateListener) {
+    public void updateAsync(@NonNull final List<? extends Group> newGroups,
+                            @Nullable final OnAsyncUpdateListener onAsyncUpdateListener,
+                            boolean detectMoves) {
         final List<Group> oldGroups = new ArrayList<>(groups);
         final int oldBodyItemCount = getItemCount(oldGroups);
         final int newBodyItemCount = getItemCount(newGroups);
 
         asyncDiffUtil.calculateDiff(newGroups,
-                new DiffCallback(oldBodyItemCount, newBodyItemCount, oldGroups, newGroups), onAsyncUpdateListener);
+                new DiffCallback(oldBodyItemCount, newBodyItemCount, oldGroups, newGroups), onAsyncUpdateListener, detectMoves);
+    }
+
+    /**
+     * Overload for {@link GroupAdapter#update(Collection, boolean)}
+     */
+    @SuppressWarnings("unused")
+    public void update(@NonNull final Collection<? extends Group> newGroups) {
+        update(newGroups, true);
     }
 
     /**
      * Updates the adapter with a new list that will be diffed on the <em>main</em> thread
      * and displayed once diff results are calculated. Not recommended for huge lists.
      * @param newGroups List of {@link Group}
+     * @param detectMoves is passed to {@link DiffUtil#calculateDiff(DiffUtil.Callback, boolean)}
```

Comment:
maybe also worth mentioning why you might want/not want to use `detectMoves` as a quick overview?